packages/shared/src/lib/kratos.ts

```diff
@@ -4,7 +4,7 @@ import { authUrl, heimdallUrl } from './constants';
 export type EmptyObjectLiteral = Record<string, never | string>;
 
 export interface SocialRegistrationFlow extends MessageEventData {
-  flow: string;
+  social_registration: boolean;
```

Comment:
What happens to the linking of accounts? That goes through here as well and it requires the flow to be present for checking any error.

Reply:
I think we can keep them both but be nullable properties.